test/common/network/connection_impl_test.cc

```diff
@@ -502,6 +503,33 @@ TEST_P(ConnectionImplTest, BindTest) {
   disconnect(true);
 }
 
+TEST_P(ConnectionImplTest, BindFailureTest) {
+  std::string address_string = TestUtility::getIpv4Loopback();
```

Comment:
Tiny nit: prefer this initialization in the ` Network::Address::IpVersion::v6` clause below.